architecture-model-generator/modules/architecture-model-generator-core/src/test/java/io/ballerina/architecturemodelgenerator/core/ServiceGeneratorTests.java

```diff
@@ -41,11 +41,6 @@ public class ServiceGeneratorTests {
 
     @Test(description = "model generation for single module projects")
     public void testSingleModuleModelGeneration() throws IOException {
-//        CompileResult compileResult = BCompileUtil.compileAndCacheBala("ballerina/single_service_sample");
-//        if (compileResult.getErrorCount() != 0) {
-//            Arrays.stream(compileResult.getDiagnostics()).forEach(System.out::println);
-//            Assert.fail("Compilation contains error");
-//        }
```

Comment:
DO we need these commented lines?

Reply:
This was already removed in the previous commit :)